account_financial_report/report/templates/general_ledger.xml

```diff
@@ -352,7 +352,41 @@
                                 </span>
                             </t>
                         </div>
-                        <div class="act_as_cell amount" style="width: 3.63%;" />
+                        <div class="act_as_cell amount" style="width: 3.63%;">
+                            <t t-if="type == 'account_type'">
+                                <t
+                                    t-set="domain"
+                                    t-value="[('account_id', '=', account['id']),
+                                             ('date', '&lt;', o.date_from)]"
+                                />
+                                <span
+                                    t-att-domain="domain"
+                                    res-model="account.move.line"
+                                >
+                                    <t
+                                        t-raw="account_or_group_item_object['init_bal']['bal_curr']"
```

Comment:
Are you sure of use t-raw?

Reply:
Yes, it is exactly the same code as the previous cell and both cells show the correct value.